johnsonandjohnson/Bodiless-JS
# feat(test-site) Add Type to all of the items in the flexbox fixes #4

## Changes


We are adding a withType to all of the components in the test flexbox
We are also removing the withAccordion which was doing the same thing

## Test Instructions


Review the flexbox component selector

See that there is a type facet with accordion Rich Text and Tout

## Related Issues

Fixes #45
